File: src/test/java/com/juunweii/PersonTest.java
package com.juunweii;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class PersonTest {

    @Test
    public void bmiTest(){
        Person person = new Person(60.6f,1.69f);
        Assertions.assertEquals(60.6f/(1.69f*1.69f) , person.bmi());

    }
}

File: src/test/kotlin/com/juunweii/PersonTest.kt
package com.juunweii.Kotlin

import com.juunweii.Human
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class PersonTest {

    @Test
    fun bmiTest(){

        val Human = Human( 60.6f , 1.7f)
        Assertions.assertEquals(60.6f/(1.7f*1.7f) , Human.bmi())
    }


}